dask_planner/src/sql.rs

```diff
@@ -479,6 +480,29 @@ impl DaskSQLContext {
             .map_err(py_parsing_exp)
     }
 
+    pub fn get_table_statistics(&self) -> Option<HashMap<String, DaskStatistics>> {
+        let mut table_statistics: HashMap<String, DaskStatistics> = HashMap::new();
+        // We use all_row_counts to ensure that we have statistics
+        // for at least one of the tables in the DaskSQLContext
+        let mut all_row_counts = 0.0;
+
+        for schema in &self.schemas {
+            for table in &schema.1.tables {
+                let dask_table = &table.1;
+                let table_name = &dask_table.table_name;
+                let statistics = &dask_table.statistics;
+                table_statistics.insert(table_name.to_string(), statistics.clone());
```

Comment:
We might end up with incorrect stats if we have tables with the same name in different schemas. A fully qualified name if applicable might be a better key.